src/openzaak/components/catalogi/admin/utils.py

```diff
@@ -204,8 +213,9 @@ def import_zaaktype_for_catalogus(
                     for old, new in besluittypen_uuid_mapping.items():
                         data = data.replace(old, str(new.uuid))
 
-                for old, new in uuid_mapping.items():
-                    data = data.replace(old, new)
+                if generate_new_uuids:
```

Comment:
I think this condition should be moved a little above and also includes `iotypen_uuid_mapping` and `besluittypen_uuid_mapping` ?

Reply:
I thought so too, but this is not the case, because even when keeping the same UUIDs for created instances, you can still link with existing instances that have different UUIDs and in that case you do want to use those mappings (same comment as https://github.com/open-zaak/open-zaak/pull/1681#discussion_r1647360243)